backend/geolibre_server/tests/test_geoparquet_detection.py

```
@@ -0,0 +1,206 @@
+    and asks the caller to pick the columns explicitly through its CSV path
+    instead. This pins the gap so it is a decision rather than a surprise.
+    """
+    source = _lon_lat_parquet(tmp_path / "lonlat_columns.parquet")
+
+    proc = _convert(source, tmp_path / "out.parquet")
+    assert proc.returncode != 0
+    assert "No geometry column found" in (proc.stdout + proc.stderr)
+
+
+def test_detects_the_geometry_column_of_the_remote_sample(
+    spatial_extension: None, sample_dataset_url: str, tmp_path: Path
```

Ruff (0.16.2)

[warning] Unused function argument: `spatial_extension`

(ARG001)